### DISCUSSION TITLE
next release

### DISCUSSION THREAD

**Discussion by GuillaumeDSM** [bmoscon/cryptofeed#918] [General]
Hello !
For a few days, we are experiencing issues with Phemex (the feed crashes on constructor call). This PR fixes it https://github.com/bmoscon/cryptofeed/pull/916
Would it be possible to release a new version to publish the fixed Phemex API please ? 
Thanks !

**Comment by GuillaumeDSM** [comment]
Hey, I would like to bump this up because this Phemex issue is completely preventing our users from using Phemex. As the crash is right at the cryptofeed exchange constructor, we can't really use a workaround and it would be sad to have to monkeypatch parts of the lib.

@bmoscon would it be possible to release the next version of cryptofeed please ?

**Reply by bmoscon** [reply]
@GuillaumeDSM alright, I can release it today

**Reply by GuillaumeDSM** [reply]
Thanks a lot @bmoscon !
